crates/matrix-sdk/src/event_cache/linked_chunk.rs

```diff
@@ -1414,7 +1414,7 @@ mod tests {
 
     #[test]
     fn test_insert_gap_at() -> Result<(), LinkedChunkError> {
-        let mut linked_chunk = LinkedChunk::<char, (), 3>::new();
+        let mut linked_chunk = LinkedChunk::<3, char, ()>::new();
```

Comment:
it reads nicely: "a linked chunk with a nominal chunk of 3 chars" :smiling_face_with_three_hearts:

Reply:
Same feeling here!